### ISSUE TITLE
[Bug]: API deprecation warnings in console upon import of Menace Under Otari

### ISSUE THREAD

**Issue by surged20** [foundryvtt/foundryvtt-premium-content#110]
### What happened?

- v10 build 284
- pf2e 4.0.0-beta1
- beginner box 2.0.0

It's just warnings, but may be scary to a lot of users.

![image](https://user-images.githubusercontent.com/71271408/188705318-0fe93bfb-506e-4c48-988b-154b4870763f.png)

### What package are you reporting this for?

Pathfinder: Beginner Box by Paizo

### Version

2.0.0

### Relevant log output

```shell
commons.js:1250 Error: You are accessing BasePackage#data which is now deprecated in favor of referencing schema fields directly on the BasePackage instance.
Deprecated since Version 10
Backwards-compatible support will be removed in Version 12
    at logCompatibilityWarning (commons.js:1239:19)
    at get data [as data] (commons.js:11230:7)
    at Object.handler (adventure.mjs:70:20)
    at async PF2EAdventureImporter._importContent (importer.mjs:83:7)
    at async PF2EAdventureImporter._updateObject (foundry.js:61343:32)
    at async PF2EAdventureImporter._onSubmit (foundry.js:5778:7)
logCompatibilityWarning @ commons.js:1250
get data @ commons.js:11230
handler @ adventure.mjs:70
commons.js:5295 You are calling DataModel#update which is renamed to DataModel#updateSource
```

**Comment by surged20** [comment]
This is actually a pf2e system issue. I can confirm it's now fixed on pf2e master with https://github.com/foundryvtt/pf2e/commit/f8d2a39bff0b1311fc83b7eca76067c00f16a298

**Comment by aaclayton** [comment]
> This is actually a pf2e system issue. I can confirm it's now fixed on pf2e master with [foundryvtt/pf2e@f8d2a39](https://github.com/foundryvtt/pf2e/commit/f8d2a39bff0b1311fc83b7eca76067c00f16a298)

Thanks for the clarification.
